styles/variables-gallium.less

```diff
@@ -262,6 +262,9 @@
 // ToggleButton
 // ---------------------------------------
 @agate-togglebutton-indicator-position: auto 33px 9px 33px;
+@agate-togglebutton-indicator-huge-position: auto 75px 33px 75px;
+@agate-togglebutton-indicator-small-position: auto 24px 6px 24px;
+@agate-togglebutton-indicator-smallest-position: auto 18px 3px 18px;
```

Comment:
since variables-base has the same values for these variables, they are not needed in gallium skin to be overwritten